webknossos/webknossos/dataset/dataset.py

```diff
@@ -1289,7 +1289,7 @@ def add_layer_from_images(
                 set(layer.bounding_box.axes).difference("x", "y", "z")
```

Comment:
See comment above. I think we should check whether additional axes only have a length of 1. Then these axe could be collapsed.